### PR TITLE
AB#69653 Shell script to change the GET to be the beta vrsion. One wo…

### PR DESCRIPTION
**Pull request by lardo-de-arnaud**
…uld run on Canvas Test and Beta after each refresh.

### CODE REVIEW

File: set-GET-beta/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.project
+*.swp
+*.env
```
Comment (nicholaswilson100) [inline set-GET-beta/.gitignore]: This is duplicate code so not needed I don't think. The same duplicate file is at the top level.  You only need 1 gitignore per repository.

File: set-GET-beta/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Switch GET tool registration to point atthe Beta version for Canvas Test and Beta - DO NOT USE ON PRODUCTION
```
Comment (nicholaswilson100) [inline set-GET-beta/README.md]: typo

Comment (nicholaswilson100) [reply set-GET-beta/README.md]: atthe

File: set-GET-beta/example.env [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# The hostname to update
+#host=institution.test.instructure.com
```
Comment (nicholaswilson100) [inline set-GET-beta/example.env:2]: These other examples of the host are fairly obvious. Do we definitely need them?

File: set-GET-beta/set-GET-beta.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+curl -X PUT 'https://universityofoxford.beta.instructure.com/api/v1/accounts/1/external_tools/9604'   \
+     -H "Authorization: Bearer ${token}"  \
+     -F 'course_navigation[text]=Oxford Groups Beta' \
+     -F 'course_navigation[label]=Oxford Groups Beta' \
+     -F 'account_navigation[text]=Oxford Groups Beta' \
+     -F 'account_navigation[label]=Oxford Groups Beta' \
+     -F 'consumer_key=ox'  \
+     -F 'name=Beta GET (Oxford Groups)' | jq
+     || (printf "Unable to set Discovery URL\n\n"; exit 1)
+
```
Comment (nicholaswilson100) [inline set-GET-beta/set-GET-beta.sh:44]: If this curl command fails but does so silently, do we want to check it's worked afterwards and send an email to Sentry if not? Or is that too much?

**Comment by nicholaswilson100** [comment]
All looks more or less fine to me , just minor comments

**Comment by nicholaswilson100** [comment]
Nice stuff

File: set-GET-beta/set-GET-beta.sh (hunk 4 above)
Comment (lardo-de-arnaud) [reply set-GET-beta/set-GET-beta.sh:44]: If the scrip[t fails then an email is sent but I realise the message is incorrect. It shouldn't be "Unable to set Discovery URL"!! If it succeeds then you getthe JSON detailing the tool config.

File: set-GET-beta/.gitignore (hunk 1 above)
Comment (lardo-de-arnaud) [reply set-GET-beta/.gitignore]: ok

File: set-GET-beta/README.md (hunk 2 above)
Comment (lardo-de-arnaud) [reply set-GET-beta/README.md]: i corrected this and the other spelling mistakes in the README.md. Ta.

File: set-GET-beta/example.env (hunk 3 above)
Comment (lardo-de-arnaud) [reply set-GET-beta/example.env:2]: it's just the standard example.env file so I'll leave it.

**Review by nicholaswilson100 — APPROVED** [review]
all looks good to me!

**Pull request merged**